snackpub/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "snackpub",
+  "version": "0.0.0",
+  "private": true,
+  "description": "The Snack PubSub system",
+  "main": "build/index",
```

Not sure if required, but I don't trust Node to resolve to the exact file 🙈 

```suggestion
  "main": "build/index.js",
```